### DISCUSSION TITLE
蓝灯中国代理网站有最新的地址吗？

### DISCUSSION THREAD

**Discussion by qilayadi** [getlantern/lantern-forum-cn#1098] [Q&A]
您好，请问蓝灯中国代理网站最新地址是什么呢？以前的永久地址：golandeng.org现在进不去了，没法买蓝灯激活码升级专业版了呢。请给下最新的蓝灯中国代理网站最新地址呢，谢谢。

**Comment by MaxMKMax** [comment]
@qilayadi 谢谢您支持蓝灯! 您好. 请您是看着两个网络连接可以下载蓝灯吗? 谢谢.
https://lantern.io/zh-Hans/download?os=windows

https://github.com/getlantern/lantern
